src/Features/Core/Portable/Completion/Providers/AbstractDocCommentCompletionProvider.cs

```diff
@@ -113,10 +113,10 @@ private CompletionItem GetItem(string name)
         return CreateCompletionItem(name);
     }
 
-    protected IEnumerable<CompletionItem> GetAttributeItems(string tagName, ISet<string> existingAttributes)
+    protected IEnumerable<CompletionItem> GetAttributeItems(string tagName, ISet<string> existingAttributes, bool addEqualsAndQuotes)
     {
         return s_attributeMap.Where(x => x.elementName == tagName && !existingAttributes.Contains(x.attributeName))
-                             .Select(x => CreateCompletionItem(x.attributeName, beforeCaretText: x.text, afterCaretText: "\""));
+                             .Select(x => CreateCompletionItem(x.attributeName, beforeCaretText: addEqualsAndQuotes ? x.text : x.text[..^2], afterCaretText: addEqualsAndQuotes ? "\"" : ""));
```

Comment:
can you wrap this a bit in a more pleasing fashion?  something like:

```
        return s_attributeMap
            .Where(x => x.elementName == tagName && !existingAttributes.Contains(x.attributeName))
            .Select(x => CreateCompletionItem(
                x.attributeName,
                beforeCaretText: addEqualsAndQuotes ? x.text : x.text[..^2],
                afterCaretText: addEqualsAndQuotes ? "\"" : ""));
```

Thanks :)

Reply:
done